src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.scss

```diff
@@ -5,7 +5,7 @@
   display: grid;
   background-color: $black;
   grid-template-columns: 228px 902px 1fr;
-  grid-template-rows: min-content 150px 1fr;
+  grid-template-rows: min-content 31px 1fr;
```

Comment:
The distance between the gene above and the buttons that I am seeing in the PR:

![image](https://user-images.githubusercontent.com/6834224/76530405-98705b80-646b-11ea-8dd4-141868621bac.png)

is smaller than in the XD:

![image](https://user-images.githubusercontent.com/6834224/76530447-a625e100-646b-11ea-8fab-8a365955d883.png)

Are you sure about this change? Isn't `align-self: end` on `geneViewTabs` better suited for this role?

Reply:
Changed it to 90px as 150px creates a big gap.